src/cryptography/hazmat/backends/__init__.py

```diff
@@ -17,3 +17,10 @@ def default_backend():
         _default_backend = backend
 
     return _default_backend
+
+
+def _get_backend(backend):
+    if backend is None:
+        return default_backend()
+    else:
+        return backend
```

Comment:
Should this warn? We intend to deprecate this eventually, right?

Reply:
I have some ideas for backends that would be much easier if the concept still exists, so I don’t want to deprecate it just yet.

Reply:
Ok.